EAD/EADSession2/Signup.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Registeration Page</title>
+	<script src="Scripts/validation.js"></script>
+</head>
+<body align="center">
+	<div>
+		<table align="center">
+			<tr>
+				<td align="left" width="60%"> <img src="Images/html.png" height="30%" /> </td>
+				<td><a href="HomePage.html">Home</a>
+					<a href="">About</a>
+					<a href="">Product</a>
+					<a href="">Gallery</a>
+					<a href="Contact.html">Contact</a>
+					<a href="Login.html">Login</a>
+				</td>
+			</tr>
+		</table>
+	</div>
+	<hr width="50%" />
+	<div>
+		<h1>Register!!!</h1>
+		<form action="Login.html" onsubmit="return validation();" name="signupform">
+			<table align="center" cellspacing="10px">
+				<tr align="left">
+					<td width="50%">First Name</td>
+					<td><input type="text" name="firstName" placeholder="Enter First Name" required></td>
```

Comment:
As we are using required property in html tag, there are two type of errors on page. UI difference (Required property error's box vs javascript error box).

Reply:
If we use 'JavaScript Validation APIs' in validations, then errors will come in same box. So please try to implement this instead of alert popups.